x-pack/plugins/upgrade_assistant/server/routes/reindex_indices/reindex_indices.ts

```diff
@@ -114,12 +115,21 @@ export function registerReindexIndicesRoutes(
             ? await reindexService.detectReindexWarnings(indexName)
             : [];
 
-          return response.ok({
-            body: {
-              reindexOp: reindexOp ? reindexOp.attributes : null,
-              warnings,
-              hasRequiredPrivileges,
+          const indexAliases = await reindexService.getIndexAliases(indexName);
+
+          const body: ReindexStatusResponse = {
+            reindexOp: reindexOp ? reindexOp.attributes : undefined,
+            warnings,
+            hasRequiredPrivileges,
+            meta: {
```

Comment:
nit: would be nice to have tests to assert that this meta field contains the right data